.github/workflows/release.yml

```diff
@@ -64,6 +64,11 @@ jobs:
       # these are the dependencies listed in release-files. Sorry for the duplication
       run: |
         nix-build --max-jobs 1 --arg officialRelease true -A moc -A mo-ide -A mo-doc -A js.moc -A js.moc_interpreter
+        echo "## checks"
+        uname -s -m
+        $(nix-build --max-jobs 1 --arg officialRelease true -A moc)/bin/moc --version
+        file $(nix-build --max-jobs 1 --arg officialRelease true -A moc)/bin/moc
+        ldd $(nix-build --max-jobs 1 --arg officialRelease true -A moc)/bin/moc || true
```

Comment:
This is just some smoke test to see the process worked before putting stuff into the tarballs.